patroni/ctl.py

```diff
@@ -630,7 +703,23 @@ def members(config_file, cluster_names, fmt, watch, w, dcs):
         dcs = get_dcs(config, cluster_name)
 
         for _ in watching(w, watch):
-            output_members(dcs.get_cluster(), cluster_name, fmt)
+            extra = dict()
```

Comment:
There is no need to build a `dict` object here and pass it into `output_members` function.
Information about `scheduled_restart` is available for every member and could be used directly from there.